src/CoreBundle/Resources/public/javascript/jsTree/customStyles/style.css

```diff
@@ -7,36 +7,30 @@
     background: unset;
 }
 
-.navigationTreeContainer .node-hidden,
-.treelegend .page-hidden {
-    color: #848484;
-}
-
-.navigationTreeContainer .activeConnectedNode,
 .treelegend .activeConnectedNode {
     font-style: normal;
-    font-weight: bold;
-    color: #008000;
+    color: #23282c;
+    background: #beebff;
+    background: -webkit-linear-gradient(top, #beebff 0%, #a8e4ff 100%);
+    background: linear-gradient(to bottom, #beebff 0%, #a8e4ff 100%);
```

Comment:
Doesn't the last line overwrite the previous ones?

Reply:
yes, if linear-gradient is supported